Disable part of test_posix.test_makedev that depends on the NODEV
macro being defined (missing on musl).

See https://github.com/python/cpython/pull/31794


diff --git a/Lib/test/test_posix.py b/Lib/test/test_posix.py
index 7ed45ac..a35c6db 100644
--- a/Lib/test/test_posix.py
+++ b/Lib/test/test_posix.py
@@ -724,7 +724,7 @@ def test_makedev(self):
             self.assertRaises((ValueError, OverflowError), posix.makedev, x, minor)
             self.assertRaises((ValueError, OverflowError), posix.makedev, major, x)
 
-        if sys.platform == 'linux':
+        if False:
             NODEV = -1
             self.assertEqual(posix.major(NODEV), NODEV)
             self.assertEqual(posix.minor(NODEV), NODEV)
